Tensile/Configs/miopen/Makefile

```diff
@@ -0,0 +1,57 @@
+P=problems
+DEEPBENCH_1x1_ALL=$P/nn/deepbench_1x1_batchN.yml $P/nn/deepbench_1x1_batch1.yml
+RESNET=$P/nn/resnet50_batch64.yml
+
+# commonly-used headers and footers:
+HEADER=boiler/header.yml
+FOOTER=boiler/library_logic_vega10_only.yml
+
+# Override TYPE as sgemm, hgemm (hgemm_hpa, dgemm, etc in future)
+TYPE=sgemm
+
+# Override these to provide custom solution level or files
+# Explore level 0..9 - higer level indicates more intensive search.
+#  Only 3,5,7 supported.
+#  Exact solutions are defined in the specified solution file.
+EXPLORE_LEVEL=3
+
+# SOLUTION_LARGE is used for large matrix type
+SOLUTION_LARGE=solutions/$(TYPE)_large_explore_$(EXPLORE_LEVEL).yml
+
+# SOLUTION_SKINNY is used for problems with skinny M or N
+SOLUTION_SKINNY=solutions/$(TYPE)_skinny_explore_$(EXPLORE_LEVEL).yml
+
+# Targets specify different collections of solutions + problems
+
+all: \
+	$(TYPE)_deepbench_gemm_nn.yaml \
+	$(TYPE)_deepbench_gemm_nt.yaml \
+	$(TYPE)_deepbench_gemm_tn.yaml
+
```

Comment:
should this have a line    $(TYPE)_deepbench_gemm_tt.yaml       ?   Also below, should there be a rule for $(TYPE)_deepbench_gemm_tt.yaml      ?